server/types.ts

```diff
@@ -6,6 +6,16 @@ export type AirtableResponse<T> =
   | { kind: "success"; records: AirtableRecord<T>[]; error?: never }
   | { kind: "error"; error: string; records?: never };
 
+type Enumerate<T extends readonly any[]> = T[number];
```

Comment:
Nice job putting this in a utility type.

Reply:
Thanks